### PR TITLE
Symfony validator integration

### LINKED ISSUE #14: Use symfony/validator on request properties
Ref #12.

### PR DESCRIPTION
**Pull request by soullivaneuh**
Closes #14.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
         "php": "^5.6 || ^7.0",
         "guzzlehttp/guzzle": "^6.0",
         "symfony/options-resolver": "^2.7 || ^3.0",
-        "greg0ire/enum": "^2.0"
+        "symfony/validator": "^2.7 || ^3.0",
+        "doctrine/annotations": "^1.2",
+        "doctrine/cache": "^1.0",
+        "greg0ire/enum": "^2.1"
```
Comment (greg0ire) [inline composer.json:20]: You could sort your deps :P

File: src/Request/AbstractRequest.php [hunk 2]
```diff
@@ -11,31 +13,45 @@
 {
     /**
      * @var int
+     *
+     * @Assert\NotBlank
+     * @Enum(class="Nexy\PayboxDirect\Enum\Activity", showKeys=true)
```
Comment (greg0ire) [inline src/Request/AbstractRequest.php:18]: Beautiful!

File: composer.json (hunk 1 above)
Comment (soullivaneuh) [inline composer.json:20]: Ha ha I knew you will say that!

Will do it after on another PR. ;-)
